includes/admin/class-sensei-extensions.php

```diff
@@ -248,31 +239,7 @@ public function add_admin_menu_item() {
 	 * @access private
 	 */
 	public function render() {
-		if ( Sensei()->feature_flags->is_enabled( 'extensions_management_enhancement' ) ) {
-			echo '<div id="sensei-extensions-page" class="sensei-extensions-page"></div>';
-			return;
-		}
-
-		// phpcs:ignore WordPress.Security.NonceVerification
-		$category = isset( $_GET['category'] ) ? sanitize_text_field( $_GET['category'] ) : null;
-
-		// phpcs:ignore WordPress.Security.NonceVerification
-		$type = isset( $_GET['type'] ) ? sanitize_text_field( $_GET['type'] ) : null;
-
-		// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable -- used in view
-		$messages   = $this->get_messages();
-		$resources  = $this->get_resources();
-		$extensions = $this->get_extensions( $type, $category );
-		// phpcs:enable
-
-		sensei_log_event(
```

Comment:
We'll want to keep this event so that we know how often this page is being viewed.

Reply:
Good catch! Added back! https://github.com/Automattic/sensei/pull/4207/commits/7c4fec65cf8d3fd6573f09d0468785065723749b

Do you think we should also change the name to differentiate the new page logs?

Reply:
I think it's better to keep it as is. We can always get that info by looking at the version number if we need to.